src/md/verification.md

```diff
@@ -0,0 +1,112 @@
+```bash
+APEX_RELEASE_CANDIDATE=apex-3.3.0-incubating
+```
+
+Verify integrity of tar.gz file:
+```bash
+gpg --verify $APEX_RELEASE_CANDIDATE-source-release.tar.gz.asc
+md5sum --check $APEX_RELEASE_CANDIDATE-source-release.tar.gz.md5
+sha512sum --check $APEX_RELEASE_CANDIDATE-source-release.tar.gz.sha
+```
+
+Verify integrity of .zip file:
+```bash
+gpg --verify $APEX_RELEASE_CANDIDATE-source-release.zip.asc
+md5sum --check $APEX_RELEASE_CANDIDATE-source-release.zip.md5
+sha512sum --check $APEX_RELEASE_CANDIDATE-source-release.zip.sha
+```
+
+## Source code verification
+You can extract source either using .tar.gz file or .zip file.
+
+### Using .tar.gz source
+Extract source using .tar.gz:
+```bash
+tar -zxvf $APEX_RELEASE_CANDIDATE-source-release.tar.gz
+```
+### Using .zip source
+```bash
+unzip $APEX_RELEASE_CANDIDATE-source-release.zip
+```
+
+Any of the two commands above will create a directory named after *apex release candidate*.
+
+Change directory:
+```bash
+cd $APEX_RELEASE_CANDIDATE
+```
+
+### Miscellaneous checks: Part I (Pre Compilation)
+1. Existence of DISCLAIMER, LICENSE, NOTICE, README.md and CHANGELOG.md files.
+Please make sure these files are present in the parent folder after extracting the zip or tar. The CHANGELOG.md file should contain change log for the current release.
+2. No unexpected binary files in the sources.
+The extracted directory should not contain any binary. Need to test this before compiling the source code.
+```bash
+find . -type f -name '*.*' | sed 's|.*\.||' | sort -u
+```
+Please make sure no binary extension is listed here. Likely candidates are .jar files.  Additionally, for Malhar, demo package files, i.e. .apa files.
+
+### Check for compilation, license headers, etc.
+
+For Apex:
+```bash
+mvn clean apache-rat:check verify -Dlicense.skip=false install
+```
+For Malhar:
+```bash
+mvn clean apache-rat:check verify -Dlicense.skip=false -Pall-modules install
+```
+Jars should be installed in your maven repository, typically in *~/.m2/repository/org/apache/apex/* directory
+
+Following step is optional and needs prerequisite 2 given above.
+```bash
+mvn verify -Papache-release -DskipTests
+```
+
+## Launch demos
+Launch few demos to make sure everything is working fine using *dtcli* utitlity. The dtcli script is present at <apex-core-folder>/engine/src/main/scripts/dtcli . If apex-core is being verified, then simply use engine/src/main/scripts/dtcli to launch the script. Otherwise, prefix it with apex-core folder path.
+
+```bash
+engine/src/main/scripts/dtcli 
+```
+You will get dtcli prompt, where demos can be launched.
+```bash
+dt> launch <demo-apk-file>
+```
+Demo apk files are typically in incubating-apex-malhar/demos directory.
+
+
+## Miscellaneous checks: Part II (Post compilation)
+1. Incubating word in artifact name.
+The artifacts generated by the build should have word incubating in it. For example, apex-common-3.3.0-incubating.jar. It should also contain the right version number. 
```

Comment:
IMO this is not valid. Should instead check the source release name and the artifacts in Maven, not in a local build directory.